src/python/grpcio_tests/tests/interop/server_ssl_cert_reload_test/_test.py

```diff
@@ -0,0 +1,411 @@
+# Copyright 2017 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import atexit
+import os
+import subprocess
+import sys
+import time
+import unittest
+import tempfile
+
+
+processes = []
+
+# Make sure we attempt to clean up any
+# processes we may have left running
+def cleanup_processes():
+    for process in processes:
+        try:
+            process.kill()
+        except Exception:
+            pass
+
+
+atexit.register(cleanup_processes)
+
+DIR = os.path.abspath(os.path.dirname(__file__))
+
+def _get_abs_path(path):
+    return DIR + '/' + path
+
+
+def run_client(cmd):
+    # run the command, wait for it to exit, and return its stdout and
+    # stderr. if there's any error in running the command, then None
+    # is returned for both
+    p = None
+    try:
+        p = subprocess.Popen(
+            cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+            env={'GRPC_VERBOSITY': 'ERROR'})
+        return p.communicate()
+    except:
+        pass
+    return None, None
+
+
+# stdout and stderr when the client successfully performs the rpc
+def assert_client_success(stdout, stderr):
+    assert stdout == 'Greeter client received: Hello, you\n', 'unexpected stdout: [{}]'.format(stdout)
```

Comment:
Please favor the assertSomething methods of unittest.TestCase over using raw assert statements in tests.

Reply:
done